backend/routes/recruiters.js

```diff
@@ -1,6 +1,7 @@
 const User = require('../models/recruiter.model')
 const router = require('express').Router()
 const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
```

Comment:
I noticed that in jobseekers there is "const authenticateToken = require('./authenticateToken');" but it isn't present here. Is there a reason why one has it and not the other?